src/assets/config_files/usecases.json

```diff
@@ -111,6 +111,156 @@
         "next": "ta_form",
         "dataprotected": true,
         "files": []
+      },
+      {
+        "title": "Preparing a protein to run an atomistic Molecular Dynamics simulation",
+        "description": "Running a Molecular Dynamics setup from a protein in PDB format to obtain a system ready to be used as input for an MD simulation. Using GROMACS MD package and the building blocks library from the BioExcel project.",
+        "experience": ["all", "power"],
+        "maturity": ["beta"],
+        "disabled": false,
+        "picture": {
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/IRB_uc1.png"
+        },
+        "next": "ta_form",
+        "dataprotected": true,
+        "files": [
+          {
+            "entryname": "Protein atomistic MD simulation setup - Setup",
+            "appid": 175,
+            "contenttype": "x-ipynb+json",
+            "extension": ".ipynb",
+            "file": "73d225d1-3590-442b-8a5a-5210ba64af19",
+            "initial": true
+          },
+          {
+            "entryname": "Protein atomistic MD simulation setup",
```

Comment:
I was able to run all the cells without any issue.
For the last cell (save the files) In my opinion, I think that probably is better to generate a zip and then dump it to the Storage because I cannot download the entire folder from the Collab and picking each of the files is kind of tiring. (@alex4200 Can we get your input for this)

Reply:
NGL scroll bug is fixed with the last version (2.7.1). This version is now fixed in the [setup notebook](https://collab.humanbrainproject.eu/#/collab/62050/nav/426321) as **_! conda install -y -c conda-forge nglview=="2.7.1"_**
zip file created containing all the generated files from the workflow. The zip file is also moved to the storage to easily download the whole project.

Reply:
I would do that if the collab had an "uncompress" option, but I couldn't find it. So I think it's better to have all the generated files to explore them, plus a zip file to easily download everything in one single shot.